zipline-loader/src/jvmMain/kotlin/app/cash/zipline/loader/internal/tink/subtle/Ed25519Constants.kt

```diff
@@ -1,135 +0,0 @@
-// Copyright 2017 Google Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-//
-////////////////////////////////////////////////////////////////////////////////
-package app.cash.zipline.loader.internal.tink.subtle
-
-import app.cash.zipline.loader.internal.tink.subtle.Field25519.expand
-import java.math.BigInteger
-
-/** Constants used in [Ed25519].  */
-internal object Ed25519Constants {
```

Comment:
... this is the old implementation that depended on BigInteger, which we don’t have.